Rye/RyeExample/Rye VC/RyeDemoViewController.swift

```diff
@@ -56,19 +58,23 @@ class RyeDemoViewController: UITableViewController {
     
     // MARK: - TableView methods
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        switch Section(rawValue: indexPath.section)! {
-        case .introduction, .insets, .text:
-            break
-        case .dismissMode:
-            updateDismissMode(for: indexPath)
-        case .viewType:
-            updateViewMode(for: indexPath)
-        case .position:
-            updatePosition(for: indexPath)
-        case .animationtype:
-            updateAnimationType(for: indexPath)
-        case .ignoreSafeAreas:
-            updateIgnoreSafeAreas(for: indexPath)
+        if let section = Section(rawValue: indexPath.section) {
```

Comment:
Why remove the force unwrap tho?

Reply:
I had a crash.. Just wanted to make sure it wouldn't happen again :)

Reply:
But if this crashes, isn't it a coding error? There's no dynamic data here, and also this is not production code, I'd say we'd rather crash than swallow the error silently here?

Reply:
It's in the demo project, yea. Just thought it was better to click twice than to crash, when you're there just to see the functionality in action. Didn't investigate too much about the crash, but sure, we can revert if that's preferred 😊👍

Reply:
I agree, that's a bit odd! I'll try and see if I can reproduce it